pkg/diagnostics/metrics.go

```diff
@@ -54,7 +101,7 @@ func InitMetrics(appID, namespace string, rules []config.MetricsRule, legacyMetr
 		return err
 	}
 
-	if err := DefaultHTTPMonitoring.Init(appID, legacyMetricsHTTPMetrics); err != nil {
+	if err := DefaultHTTPMonitoring.Init(appID, metricSpec.GetEnabled()); err != nil {
```

Comment:
I think this is the correct replacement?  I just figured it'd be easier to pass the entire `metricsSpec`.

Reply:
Nope.  I should have gone back and checked.